docs/development/release_management.rst

```diff
@@ -222,10 +222,22 @@ Release Process
 Post-Release
 ------------
 
-After the release, carefully monitor the FPF support portal (or ask those that have access to
-monitor) and SecureDrop community support forum for any issues that users are
-having.
-
-Finally, in a PR back to develop, cherry-pick the release commits (thus ensuring a consistent
-changelog in the future) and bump the version numbers
-in preparation for the next release (this is required for the upgrade testing scenario).
+Now it's time to backport the changelog from the release branch and bump the SecureDrop version so
+that it's ready for the next round of QA testing.
+
+We backport the changelog by cherry-picking any commits that modified `changelog.md` during the
```

Comment:
These docs are reStructuredText, so you'll need to use 
````rst
``this syntax``
````
for inline code fromatting and
````rst
`this syntax <https://example.com/>`_.
````
to make external links. (There are other ways to make links; see the docs themselves for examples.)